src/loki/search.cc

```diff
@@ -299,7 +290,7 @@ struct bin_handler_t {
                       PathLocation& correlated,
                       std::vector<PathLocation::PathEdge>& filtered) {
     // we need this because we might need to go to different levels
-    auto score = candidate.point.Distance(location.latlng_);
```

Comment:
this was wrong! what this says is store the distance from the input location to the snapped location along the edge. the thing, this is the function that correlates a result to a node, ie a node_snap. which means we really should be measuring the distance from the input location to the node_ll.

ive changed it to do so, which made the 32bit build fail. it failed because the precision of the node_ll that it gets out of the graph tile made it such that the node_snap measured about 9cm away from the input location, instead of 0 which it should have been.